packages/owl-ui/src/OwlPlugin.ts

```diff
@@ -0,0 +1,25 @@
+import { QWeb } from '@odoo/owl';
+import { Env } from '@odoo/owl/dist/types/component/component';
+import { JWPluginConfig, JWPlugin } from '../../core/src/JWPlugin';
+import { OwlDomRenderer } from './ui/OwlDomRenderer';
+import JWEditor, { Loadables } from '../../core/src/JWEditor';
+import { Renderer } from '../../plugin-renderer/src/Renderer';
+
+export interface OwlUIEnv extends Env {
+    editor: JWEditor;
+}
+
+export class Owl<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T> {
+    readonly loadables: Loadables<Renderer> = {
+        renderers: [OwlDomRenderer],
+    };
+
+    owlEnv: OwlUIEnv = {
+        qweb: new QWeb(),
+        editor: this.editor,
+    };
+
+    addTemplates(xmlstr: string | Document): void {
```

Comment:
To add use ressource from another plugin we use the loadable mechanism.

Reply:
:+1: